polyfill/README.md

```diff
@@ -81,6 +81,18 @@ io.observe(someTargetElement);
 
 **Note:** the `POLL_INTERVAL` property must be set prior to calling the `.observe` method, or the default configuration will be used.
 
+**Dom changes through MutationObserver**
```

Comment:
s/Dom/DOM/

I'd remove the mention of `MutationObserver` in the title here since it's primarily an implementation detail (after thinking about it a bit more, arguably the name could change as well).

Maybe change it to something like "Ignoring DOM changes".